projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -64,18 +69,26 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
   private openNodes: HTMLElement[] = [];
   private subscriptions = new Subscription();
   private resize = new EventEmitter();
+  protected arrowControls: Subject<KeyboardEvent> = new Subject();
 
   @HostListener('window:resize')
   onResize() {
     this.resize.next(undefined);
   }
 
+  @HostListener('document:keyDown.arrowUp', ['$event'])
+  @HostListener('document:keyDown.arrowDown', ['$event'])
+  onArrow(e: KeyboardEvent) {
+    this.arrowControls.next(e);
+  }
+
   constructor(
     private router: Router,
     private renderer: Renderer2,
     private elemRef: ElementRef,
     protected hamburgerMenuService: HamburgerMenuService,
-    protected winRef: WindowRef
+    protected winRef: WindowRef,
+    protected featureConfigService: FeatureConfigService
```

Comment:
We need to make this an optional service because it is possible for services not to be provided when users extend this library. Users will need to make sure it is included to enable breaking change features/fixes.

eg `Optional() protected featureConfigService?: FeatureConfigService`

Reply:
done